Sources/CacheAdvance/CacheReader.swift

```diff
@@ -60,11 +60,17 @@ final class CacheReader {
             return message
 
         case .emptyRead:
+            guard !previousReadWasEmpty else {
+                // If the previous read was also empty, then the file has been corrupted.
+                // Two empty reads in a row means that offsetInFileAtEndOfNewestMessage is incorrect.
+                // This inconsistency likely occurred because a crash occurred during a message write.
```

Comment:
```suggestion
                // This inconsistency likely occurred because a crash happened during a message write.
```

Let's not be redundant with "occurred" 😁

Reply:
```suggestion
                // This inconsistency likely is likely due to a crash occurring during a message write.
```